tasmota/tasmota_xsns_sensor/xsns_63_aht1x.ino

```diff
@@ -56,12 +59,15 @@
 #define AHT_TEMPERATURE_OFFSET  50
 #define KILOBYTE_CONST          1048576.0f
 
-#define AHT1X_CMD_DELAY      40
-#define AHT1X_RST_DELAY      30
+#define AHT1X_CMD_DELAY      20
+#define AHT1X_RST_DELAY      10
```

Comment:
Any view on the previous values? Did you check with AHT10/AHT20? Did you check data sheets?

Reply:
Personally, I have only tested the performance of the AHT20 and AHT30. They work. I do not have other versions of these sensors. The delays are taken from the datasheet for AHT20. In AHT30 datasheet initialization requires 5ms delay, and there is nothing about softset. The measurement time is the same for all three versions - 75ms. AHT10 datasheet requires less than 20ms for initialization after power on. But it happens at power-on and it is definitely faster than the ESP itself is initialized.